bindings/python/test/coordinate/spherical/test_lla.py

```diff
@@ -10,146 +10,287 @@
 Angle = physics.units.Angle
 Length = physics.units.Length
 LLA = physics.coordinate.spherical.LLA
+Spherical = physics.environment.objects.celestial_bodies.earth.models.Spherical
+WGS84_EGM96 = physics.environment.objects.celestial_bodies.earth.models.WGS84_EGM96
 
 
-def test_coordinate_spherical_lla_constructors():
-    # Main Constructor
-    latitude: Angle = Angle(30.0, Angle.Unit.Degree)
-    longitude: Angle = Angle(30.0, Angle.Unit.Degree)
-    altitude: Length = Length(1000.0, Length.Unit.Meter)
+@pytest.fixture
+def latitude_float() -> float:
```

Comment:
Proposed improved naming:

```suggestion
def latitude_deg() -> float:
```